dataminer-overview/Pricing/Pricing_Commercial_Models.md

```diff
@@ -55,7 +55,7 @@ Usage-based services are charged based on the actual monthly usage of services,
 |---------|----------------------------|----------------------|
 | System capacity | Server-based licenses, each with fixed capacity tiers of managed objects: 5, 10, 25, 50, 100, 250, 500 , 1000. | Volume-based, per managed object or monitored metric per month. Independent from the number of nodes deployed. |
 | Failover | Server-based license | Included. Users can deploy additional nodes for Failover without paying additional fees since billing is based on the active number of managed objects or monitored metrics in a month. |
-| Optional functions | Licensed per node: Correlation and Automation engines, Dashboards app, Low-code Apps, Process Automation, DataMiner Object Models (DOM), Spectrum Analysis, and Infrastructure Discovery & Provisioning | Not applicable. All optional modules are included. |
+| Optional functions | Licensed per node: Correlation and Automation engines, Dashboard, No-code and Low-code Apps, Process Automation, DataMiner Object Models (DOM) and Spectrum Analysis | Not applicable. All optional modules are included. |
```

Comment:
The current contraction of "Dashboard, No-code and Low-code Apps" could be confusing, so I'd suggest to change this as follows (note the use of the [Oxford comma ](https://www.grammarly.com/blog/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/) as well):

```suggestion
| Optional functions | Licensed per node: Correlation and Automation engines, Dashboard app, No-Code Apps, Low-Code Apps, Process Automation, DataMiner Object Models (DOM), and Spectrum Analysis | Not applicable. All optional modules are included. |
```